### PR TITLE
chore(deps): update deps

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### Motivation / Use-Case

Update.

### Breaking Changes

no

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,22 +89,21 @@
     "execa": "^4.0.0",
     "file-loader": "^6.0.0",
     "html-loader": "^1.1.0",
-    "html-webpack-plugin": "^4.2.0",
+    "html-webpack-plugin": "^4.3.0",
     "husky": "^4.2.5",
-    "jest": "^25.4.0",
+    "jest": "^26.0.1",
     "jest-junit": "^10.0.0",
-    "jquery": "^3.5.0",
```
Comment (hiroppy) [inline package.json:96]: We don't need jquery.

File: test/cli/cli.test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('CLI', () => {
     testBin('--progress')
       .then((output) => {
         expect(output.exitCode).toEqual(0);
-        expect(output.stderr).toContain('0% compiling');
+        expect(output.stderr).toContain('100%');
```
Comment (hiroppy) [inline test/cli/cli.test.js:14]: This change is for webpack@5.

https://dev.azure.com/webpack/webpack-dev-server/_build/results?buildId=9700&view=logs&j=34838ad4-6e10-570d-19ad-fabd2c8dff4b&t=e0d4cf5c-23bc-5ff5-8c16-7ee6929a9592

File: test/server/progress-option.test.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ describe('progress', () => {
         let foundProgress = false;
         let foundProfile = false;
         calls.forEach((call) => {
-          if (call[0].includes('0% compiling')) {
+          if (call[0].includes('2%')) {
```
Comment (hiroppy) [inline test/server/progress-option.test.js:32]: The previous webpack@next of beta was able to output 0% but the new version can't output this. I'll investigate this but anyway, this PR is okay.(progress feature can work).

File: test/cli/cli.test.js (hunk 2 above)
Comment (evilebottnawi) [reply test/cli/cli.test.js:14]: /cc @sokra

**Pull request merged**